context-py/tests/test_parse_compressed_codeowners.py

```diff
@@ -0,0 +1,7 @@
+def test_parse_meta_from_tarball():
+    from context_py import codeowners_parse
+
+    codeowners_text = """* @trunk/test"""
+    codeowners = codeowners_parse(str.encode(codeowners_text))
```

Comment:
No need to encode the string here, Python has a byte-string notation

```suggestion
    codeowners_text = b"""* @trunk/test"""
    codeowners = codeowners_parse(codeowners_text)
```